framework/src/controller/migrations/migration.js

```diff
@@ -243,8 +264,8 @@ class Migration extends BaseEntity {
 	 * @returns {Promise<Array<Object>>}
 	 * Promise object that resolves with an array of objects `{id, name, path, file}`.
 	 */
-	readPending(lastMigrationId) {
-		const updatesPath = path.join(__dirname, '../sql/migrations/updates');
+	readPendingInternalMigrations(lastMigrationId) {
+		const updatesPath = path.join(__dirname, './sql/updates');
```

Comment:
All these updates for schema does not belongs to framework. These are all part of the `chain` module. Without chain module, there is no database schema related to framework. So move these all migrations to `chain` migrations. 

Afterwards framework would only need one table to exists `migrations`. That you can check/create without any migration. As its the prerequisite for running the migrations.

Reply:
There is no `chain` migration here. All sql files in this folder contains framework-related migrations only.